rendered/manifests/tests/splunk-connect-for-snmp/templates/traps/service.yaml

```diff
@@ -12,11 +12,14 @@ metadata:
     app.kubernetes.io/managed-by: Helm
   annotations:
     metallb.universe.tf/allow-shared-ip: "splunk-connect"
+    metallb.universe.tf/loadBalancerIPs: 10.202.6.213
```

Comment:
I don't how it's working but it's strange that for test we using hardcoded IP of LB

Reply:
this is just rendered examples. For the tests the ip is taken when building the environment.